docs/_static/vdr/v2.yaml

```diff
@@ -0,0 +1,268 @@
+openapi: "3.0.0"
+info:
+  title: Nuts Verifiable Data Registry API spec
+  description: API specification for the Verifiable Data Registry
+  version: 2.0.0
+  license:
+    name: GPLv3
+servers:
+  - url: http://localhost:1323
+paths:
+  /internal/vdr/v2/did:
+    post:
+      summary: Creates a new Web DID
+      description: |
+        A new web DID is created. Only the DID is returned. The DID document can be retrieved using the DID resolution endpoint.
+        A single key pair is generated and added to the DID document. The key pair is used for all verificationMethods.
+        No services are added. The PublicURL needs to be configured.
+
+        error returns:
+        * 412 - PublicURL is not configured
+        * 500 - An error occurred while processing the request
+      operationId: "createDID"
+      tags:
+        - DID
+      responses:
+        "200":
+          description: "New DID has been created successfully. Returns the DID document."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DIDDocument'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vdr/v2/did/{did}:
```

Comment:
this only resolves did:web DIDs owned by the local node? I think we already have `/iam/{did}` which essentially does the same?

Reply:
We could limit it to own docs and not do a resolve but a simple GET?